src/lib.rs

```diff
@@ -569,6 +613,8 @@ pub struct PerCrateAnalysis {
     ref_spans: HashMap<Id, Vec<Span>>,
     globs: HashMap<Span, Glob>,
     impls: HashMap<Id, Vec<Span>>,
+    per_fn_borrows: HashMap<Id, BorrowData>,
+    fn_span_lookup_tree: BTreeMap<SearchSpan, Id>,
```

Comment:
It's not really enforced at all. The only reason it's working is because I only insert `SearchSpan`s of kind `InTree`into the `BTreeMap`. I'll add it back in since that will prevent any errors in the future when someone else tries to modify the code.